src/services/item/plugins/app/appData/schemas.ts

```diff
@@ -87,6 +112,7 @@ export const getForOne = {
     additionalProperties: false,
   },
```

Comment:
The Querystring object is not typed using typebox, should it be added ? And documented ?